### PR TITLE
Add docs for HAVING + GROUP BY clause

### PR DESCRIPTION
**Pull request by sbs2001**
This PR adds an example on how to have  `HAVING + GROUP BY ` clause using ent.

### CODE REVIEW

File: doc/md/aggregate.md [hunk 1]
```diff
@@ -84,4 +84,57 @@ func Do(ctx context.Context, client *ent.Client) {
 		}).
 		Scan(ctx, &users)
 }
+```
+
+## Having + Group By
+
+This example requires custom SQL modifiers. See how to enable them [here](features.md#custom-sql-modifiers)
```
Comment (yonidavidson) [inline doc/md/aggregate.md:91]: Please change URL to https://entgo.io/docs/feature-flags/#custom-sql-modifiers

File: doc/md/aggregate.md [hunk 2]
```diff
@@ -84,4 +84,57 @@ func Do(ctx context.Context, client *ent.Client) {
...
+					sql.Raw(sql.Max(user.FieldAge)),
+				),
+			)
+		}
+	).ScanX(ctx, &users)
+}
+
+```
+
+**Note:** The `sql.Raw` is crucial to have. It  tells the predicate that `sql.Max` or whatever wrapped function is not an arguement.
```
Comment (yonidavidson) [inline doc/md/aggregate.md:132]: you have double space after 'it'.
No need to "whatever " I think it's clear enough.

File: doc/md/aggregate.md (hunk 1 above)
Comment (sbs2001) [reply doc/md/aggregate.md]: fixed

File: doc/md/aggregate.md (hunk 2 above)
Comment (sbs2001) [reply doc/md/aggregate.md]: fixed

**Comment by sbs2001** [comment]
@yonidavidson   Thanks for the review. I made the suggested changes, please re-review.

File: doc/md/aggregate.md [hunk 3]
```diff
@@ -84,4 +84,57 @@ func Do(ctx context.Context, client *ent.Client) {
...
+					sql.Raw(sql.Max(user.FieldAge)),
+				),
+			)
+		}
+	).ScanX(ctx, &users)
+}
+
+```
+
+**Note:** The `sql.Raw` is crucial to have. It tells the predicate that `sql.Max` is not an arguement.
```
Comment (yonidavidson) [inline doc/md/aggregate.md:132]: [](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)[](https://github.com/)**Note:** The `sql.Raw` is required to indicate the predicate that `sql.Max` is not an arguement.
 

Reference an issue, pull request, or discussion

**Review by a8m — APPROVED** [review]
💯

**Pull request merged**